project4/project/ebs/models.py

```diff
@@ -85,3 +85,5 @@ class BlogFile(models.Model):
     blog = models.ForeignKey(Blog, blank=True, on_delete=models.CASCADE)
     def __unicode__(self):
         return self.blog.title
+   
+    
```

Comment:
EOF (This is repeating in all PRs, take care of this without fail for upcoming PRs.)